resources/views/service/dashboard.blade.php

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript">
         function printCentreRegistrations()
         {
-            window.open('{{ URL::route("service.centre.registrations.print", ["centre" => $centre_id]) }}');
+            window.open('{{ URL::route("service.registrations.print") }}');
```

Comment:
What about the collection sheet ? For centres with that pref?

Reply:
service.centre.registrations ? Is that the one that should be here if a centre opted for print_pref collection?

Reply:
The switching business is part of another card methinks